themes/digital.gov/src/js/edit-this.js

```diff
@@ -0,0 +1,52 @@
+jQuery(($) => {
+  function enableEditThis() {
+    // eslint-disable-next-line func-names
+    $("*[data-edit-this]").each(function () {
+      const filepath = $(this).data("edit-this");
+      //
+      // Disabling no-undef & camelcase because these are defined in templates.
+      //
+      // eslint-disable-next-line no-undef, camelcase
+      const editLink = `<a class="edit-this-btn" href="https://github.com/${git_org}/${git_repo}/edit/${branch}/content/${filepath}" title="edit this" target="_blank"><span>edit</span></a>`;
+
+      $(this).addClass("edit-this").append(editLink);
+    });
+  }
+  function disableEditThis() {
+    // eslint-disable-next-line func-names
+    $("*[data-edit-this]").each(function () {
```

Comment:
@mejiaj Should follow the `es-lint-disable-next-line` rule like the function before it and added the function keyword.

Reply:
Good catch, thanks.